src/query/src/dist_plan/commutativity.rs

```diff
@@ -33,7 +33,7 @@ pub struct Categorizer {}
 impl Categorizer {
     pub fn check_plan(plan: &LogicalPlan) -> Commutativity {
         match plan {
-            LogicalPlan::Projection(_) => Commutativity::Commutative,
+            LogicalPlan::Projection(_) => Commutativity::Unimplemented,
```

Comment:
Create an issue to track it.

Reply:
https://github.com/GreptimeTeam/greptimedb/issues/1702